aspnetcore/security/authentication/windowsauth.md

```diff
@@ -172,6 +172,36 @@ app.UseAuthentication();
 
 For more information on middleware, see <xref:fundamentals/middleware/index>.
 
+Kerberos authentication does not provide any role information for an authenticated users. In order to add role and group information to a kerberos user the kerberos  authentication handler must be configured to retreive the roles from an LDAP domain. The most basic configuration only specifies an LDAP domain to query against and will use the authenticated user's context to query the LDAP domain;
```

Comment:
Even though this API would "work" on Windows, it's mostly only relevant to Linux scenarios.

Reply:
It's implied further up, @scottaddie should it get repeated down here? It's your doc really